tests/integration/security/jwt_test.go

```diff
@@ -66,251 +63,290 @@ func TestRequestAuthentication(t *testing.T) {
 			ctx.Config().ApplyYAMLOrFail(t, ns.Name(), jwtPolicies...)
 			defer ctx.Config().DeleteYAMLOrFail(t, ns.Name(), jwtPolicies...)
 
-			var a, b, c, d, e, f echo.Instance
-			echoboot.NewBuilder(ctx).
-				With(&a, util.EchoConfig("a", ns, false, nil, nil)).
-				With(&b, util.EchoConfig("b", ns, false, nil, nil)).
-				With(&c, util.EchoConfig("c", ns, false, nil, nil)).
-				With(&d, util.EchoConfig("d", ns, false, nil, nil)).
-				With(&e, util.EchoConfig("e", ns, false, nil, nil)).
-				With(&f, util.EchoConfig("f", ns, false, nil, nil)).
-				BuildOrFail(t)
+			aSet := apps.A.Match(echo.Namespace(ns.Name()))
```

Comment:
+1 to Johns comment below. Are all apps here deployed to a specific namespace?

Reply:
Same above. This PR seems to be more stable and smaller, which can be merged earlier.

Reply:
I am not convinced this is right even inthe context of the other PR. But its not bad here